pkg/controller/podautoscaler/horizontal_test.go

```diff
@@ -704,20 +683,6 @@ func TestScaleUpCMUnreadyNoScaleWouldScaleDown(t *testing.T) {
 	tc.runTest(t)
 }
 
-func TestDefaultScaleDown(t *testing.T) {
```

Comment:
We don't have implicit defaulting any more -- it's done a the API level (defaulting for v2alpha1, and conversion for v1 for reasons of method signatures and JSON unmarshalling).